### DISCUSSION TITLE
Global Effects

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#556] [Help] [answered]
So... I was wondering the if some addons could effect people who don't have scratch addons, like adding more blocks to the editor, and then people who view the project with the addon can see the effects.
This idea is absurd.

**Comment by GrahamSH-LLK** [comment, marked as answer]
>This idea is absurd.

Ha! No, that wouldn't be possible. Unless scratch supports the block, Scratch has no idea what to do with it.

**Comment by Explosion-Scratch** [comment]
This would be really cool. It'd have to be supported somewhat by scratch
itself though, examples of stuff that would work are the while loop
obsolete block, the loud boolean, the for i in range block and some hacked
blocks by qucchia.
